explorer/mempool.go

```diff
@@ -76,25 +90,42 @@ func (exp *explorerUI) mempoolMonitor(txChan chan *NewMempoolTx) {
 			}
 		}
 
+		// get txraw
+		txraw, err := exp.blockData.DecodeRawTransaction(ntx.Hex)
```

Comment:
Really `DecodeRawTransaction` shouldn't even exist.  I think it was used in explorer/websockethandlers.go because it actually needed the struct in a dcrjson format.  But here all that is needed is the `Vin` slice, the number of vins and vouts, and the tx hash (`Txid`).  In general, we should use `txhelpers.MsgTxFromHex` so we do not have to make an RPC.

But in the function, that was already done way above.  So you just need to get rid of lines 93-107 and edit the MempoolTx assignment to:

```go
tx := MempoolTx{
	TxID:      msgTx.TxHash().String(),
	Hash:      hash,
	Time:      ntx.Time,
	Size:      int32(len(ntx.Hex) / 2),
	TotalOut:  txhelpers.TotalOutFromMsgTx(msgTx).ToCoin(),
	Type:      txhelpers.DetermineTxTypeString(msgTx),
	VoteInfo:  voteInfo,
	VinCount:  len(msgTx.TxIn),
	VoutCount: len(msgTx.TxOut),
	Coinbase:  blockchain.IsCoinBaseTx(msgTx),
}
```